homeassistant/components/overkiz/button.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
I am in doubt if https://github.com/home-assistant/core/blob/ec32b934a56d2c3bac3a082e508cb4b8d73e9739/homeassistant/components/overkiz/const.py#L70 should be removed as well. Not sure if we there are other GarageDoor devices that support all IO commands.

Is this PR the preparation for the renewed cover implementation? 😍

Reply:
Taken from my parent’s configuration. As you can see some GarageDoor can have `open` and `close` commands.
```JSON
        {
          "creationTime": 1666953304000,
          "lastUpdateTime": 1666953304000,
          "label": "** ** ** *",
          "deviceURL": "ogp://****-****-7672/4029677",
          "shortcut": false,
          "controllableName": "ogp:GarageDoor",
          "definition": {
            "commands": [
              {
                "commandName": "close",
                "nparams": 0
              },
              {
                "commandName": "goToAlias",
                "nparams": 1
              },
              {
                "commandName": "identify",
                "nparams": 0
              },
              {
                "commandName": "open",
                "nparams": 0
              },
              {
                "commandName": "saveAlias",
                "nparams": 1
              },
              {
                "commandName": "setName",
                "nparams": 1
              },
              {
                "commandName": "setNodeDefinition",
                "nparams": 3
              },
              {
                "commandName": "stop",
                "nparams": 0
              }
            ],
            "states": [
              {
                "type": "DiscreteState",
                "values": [
                  "available",
                  "unavailable"
                ],
                "qualifiedName": "core:AvailabilityState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:ErrorState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:ErrorsState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:FirmwareHashState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:FirmwareRevisionState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:NameState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:NewFirmwareAvailableState"
              },
              {
                "type": "DiscreteState",
                "values": [
                  "closed",
                  "open"
                ],
                "qualifiedName": "core:OpenClosedState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:ReachedAliasesState"
              },
              {
                "type": "DataState",
                "qualifiedName": "core:RemovableState"
              },
              {
                "type": "ContinuousState",
                "qualifiedName": "core:SignalQualityState"
              }
            ],
            "dataProperties": [],
            "widgetName": "DynamicGarageDoor",
            "uiProfiles": [
              "StatefulOpenClose",
              "OpenClose"
            ],
            "uiClass": "GarageDoor",
            "qualifiedName": "ogp:GarageDoor",
            "type": "ACTUATOR"
          },
```

> Is this PR the preparation for the renewed cover implementation? 😍
Well, it can help indeed :)